src/main/java/org/mule/extension/http/internal/multipart/HttpMultipartTransformer.java

```diff
@@ -6,40 +6,31 @@
  */
 package org.mule.extension.http.internal.multipart;
 
-import static java.lang.Math.toIntExact;
-import static java.util.stream.Collectors.toList;
-import static org.mule.runtime.api.metadata.DataType.BYTE_ARRAY;
-import org.mule.runtime.api.message.MultiPartPayload;
-import org.mule.runtime.api.metadata.TypedValue;
-import org.mule.runtime.api.transformation.TransformationService;
-import org.mule.runtime.core.api.message.PartAttributes;
 import org.mule.runtime.http.api.domain.entity.multipart.HttpPart;
 
-import java.util.Collection;
-
 /**
- * Handles the transformation of {@link MultiPartPayload} to an {@link HttpPart} collection.
+ * Handles the transformation of MultiPartPayload to an {@link HttpPart} collection.
  *
  * @since 1.0
  */
 public class HttpMultipartTransformer {
```

Comment:
I want to leave all the code and classes related to the tests so that migrating them is easier. Otherwise I'll have to compare them to some older branch to figure out what each one did. The ignore associated to this tests is the task regarding their migration.